client-core/src/service/hd_key_service.rs

```diff
@@ -55,7 +55,7 @@ where
             ));
         }
 
-        let seed = Seed::new(mnemonic, passphrase.unsecure());
+        let seed = Seed::new(mnemonic, "");
```

Comment:
you can also leave a TODO note for the "advanced/optional recovery" seeding option

Reply:
ok